doc/developer/platform/ux.md

```diff
@@ -772,6 +772,15 @@ running will be hidden from users. Any log messages in `materialized` today that
 are meant for end-users will need to instead make their messages available via
 tables or sources in the system catalog.
 
+### Correctness
+
+Materialize aims to maintain three correctness guarantees.
+
+1. *Materialize is serializable (or strictly serializable) against all internal reads and writes.*
+2. *Materialize respects the explicit or implied event order of its sources.*  In practice this means we reclock sources without transactional semantics (Kafka streams) to reflect the order in which the source emits each event and for sources that do have transactional semantics (Postgres, Debezium) we reclock events to respect the transactional boundaries.
```

Comment:
I admit im less of an expert on #2 than #1, but I think we might want a light definition of "reclock" here?

Reply:
@guswynn - I agree.  Added.